client/templates/course/lesson/lesson.js

```diff
@@ -7,5 +7,12 @@ Template.lesson.helpers({
     var lesson = Lessons.findOne({_id: lessonID});
 
     return lesson;
+  },
+
+  'activeQuiz': function(){
+    var quizId = activeQuizID.get();
+
+    var quiz = Quiz.convertToQuizObject(Quizzes.findOne({ _id: quizId }));
```

Comment:
This may be unnecessary, as results of `Collection.findOne()` are already objects.